.github/workflows/check-dist-sha256.yml

```diff
@@ -0,0 +1,51 @@
+name: Check dist sha256
+# Validates whether the dist folder is as expected.
+
+on:
+  push:
+    branches:
```

Comment:
do we need on `push`? Why not jut on pull_request and workflow_dispatch?

Reply:
I have removed the push.